src/modal/modal.js

```diff
@@ -414,12 +414,13 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.stackedMap'])
           backdrop: modal.backdrop,
           keyboard: modal.keyboard,
           openedClass: modal.openedClass,
-          windowTopClass: modal.windowTopClass
+          windowTopClass: modal.windowTopClass,
+          container: modal.container
         });
 
         openedClasses.put(modalBodyClass, modalInstance);
 
-        var body = $document.find('body').eq(0),
+        var container = $document.find(modal.container).eq(0),
```

Comment:
Sorry.  Support what?  jqLite supports both `.eq()` and `.append()` with no strings attached (according to [this](https://docs.angularjs.org/api/ng/function/angular.element)).